### DISCUSSION TITLE
New features branch.

### DISCUSSION THREAD

**Discussion by berrystephenw** [berrystephenw/OctoText#34] [Ideas]
I will be starting to work on some of the new features this weekend. If you are interested in testing them out that would be helpful and advance the release of the changes quicker. I'll be publishing them on the rc branch (release candidate). 

The release candidate branch can be easily switch to in Octoprint without having to spend any time coming back to github, installing by URL or downloading a zip file. You can find it here:
![image](https://user-images.githubusercontent.com/44931130/113370497-0830dd00-9332-11eb-95e8-20a8978968c4.png)

Please don't switch yet, there isn't a branch created at the moment but I'll be setting it up soon. Once it's ready I'll announce it in this discussion.

**Comment by berrystephenw** [comment]
# We are live on the RC branch! Testers requested!

I've added a few new features that have been requested:

* two new notifications: Print paused, resumed. might be able to be used for filament changes.
* Changed the test icon and allowed it to be removed in the settings page.
* Added printer name to notifications.
* Fixed the 'None' subject line sent in some conditions.

Before I push this out to everyone - it would be great if I could get some testers. Otherwise it will take longer for me to release it since I'll have to test each feature serially. If you are interested, simply change the release channel as shown above and you should get a notice that an upgrade is available. Thanks!

![image](https://user-images.githubusercontent.com/44931130/113520061-8ffb2f00-955e-11eb-84a2-249844040ade.png)

**Reply by garyriet** [reply]
Sweet. I’ll setup a few printers to use the Rc

**Reply by garyriet** [reply]
Have it working on two printers so far so good! Thanks!

**Reply by TwomD** [reply]
First: Thanks for the Plugin ! 
this is the Plugin i was waiting for :)

now: 
i did try the RC branch but it didn't work for me ( at least not the way i want to use it ;) )

My setup: Ender 3 Pro / Raspberry pi 3b with latest octopi
right now im trying to use it in combination with the "Multi Color" Plugin.
this plugin injects something in the Gcode that pauses the print at a specified Layer Hight so the user can change Filament.
then it waits for the user to change the Filament ( and if you forget about it, it waits, and waits, ..... while cooking the fillament in the nozzle -_- )
after you change the Filament you hit the Resume Button in Octoprint and it resumes Printing.

the code it injects is:
"
M117 Change filament
M25
"

at least for my ender 3, usually it would use M0 instead of M25 but that dosent work for the ender 3 ( maybe thats the Problem ? )

the rest of the email Notifications work ( when it starts / is finished or when it crashes )

**Reply by berrystephenw** [reply]
Hi @TwomD - so I've added notifications for paused prints, what you described should work since it pauses the printer. Can you send me your octoprint.log file? I can probably model something similar on my Prusa - the slicer allows you to add pauses at certain layers easily.

**Reply by TwomD** [reply]
with the new Rc that Released Today, its Working for me now :) 
thanks 👍

**Comment by berrystephenw** [comment]
The RC branch 0.2.0rc5 is the end of the line for release candidate 0.2.0 and 0.2.0 is now available on the stable branch. I will start a new release candidate as necessary for any new features and bug fixes.
